design-system/materials/internals/definition.yaml

```diff
@@ -427,7 +440,7 @@ decisionGroupsByTheme:
           choice: color-error
         font-color-for-input-when-readonly:
           description: ''
-          choice: color-neutral-60
+          choice: color-neutral-40
```

Comment:
Here we are changing the value for the current theme and we're not supposed to change anything in that theme.
Is this a new UX requirement?

Reply:
Updated [38d3d54](https://github.com/commercetools/ui-kit/pull/2347/commits/38d3d543be3051df492edcd0d5722413e61a5499)